redislock.go

```diff
@@ -236,24 +237,24 @@ func (r linearBackoff) NextBackoff() time.Duration {
 type limitedRetry struct {
 	s RetryStrategy
 
-	cnt, max int
+	cnt, max int32
 }
 
 // LimitRetry limits the number of retries to max attempts.
-func LimitRetry(s RetryStrategy, max int) RetryStrategy {
+func LimitRetry(s RetryStrategy, max int32) RetryStrategy {
```

Comment:
Would you mind leaving this as an `int`? I prefer not to introduce API changes

Reply:
I am not aware of breaking the API, it's fixed now.